docs/content/basic/practices.rst

```diff
@@ -44,59 +40,48 @@ but meant to provide a few places to look when you are developing and would
 like to check that the code you wrote satisfies the assumptions you think it
 should.
 
-All tests inherit from :code:`unittest` which is a part of core python.
-Checkout the docs on `unittest
-<https://docs.python.org/2.7/library/unittest.html>`_.
+Testing is performed with :code:`pytest` which is available through PyPI.
+Checkout the docs on `pytest <https://docs.pytest.org/>`_.
 
 
 Compare with known values
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In a simple case, you might know the exact value of what the output should be
-and you can :code:`assert` that this is in fact the case. For example, in
-`test_basemesh.py
-<https://github.com/simpeg/discretize/blob/main/tests/base/test_basemesh.py>`_,
+and you can :code:`assert` that this is in fact the case. For example,
 we setup a 3D :code:`BaseRectangularMesh` and assert that it has 3 dimensions.
 
 .. code:: python
 
-    import unittest
-    import sys
     from discretize.base import BaseRectangularMesh
     import numpy as np
 
-    class TestBaseMesh(unittest.TestCase):
-
-        def setUp(self):
-            self.mesh = BaseRectangularMesh([6, 2, 3])
+    mesh = BaseRectangularMesh([6, 2, 3])
 
-        def test_meshDimensions(self):
-            self.assertTrue(self.mesh.dim, 3)
+    def test_meshDimensions():
+        assert mesh.dim == 3
 
-The class inherits from :code:`unittest.TestCase`. When running the tests, the
-:code:`setUp` is run first, in this case we attach a mesh to the instance of
-this class, and then all functions with the naming convention :code:`test_XXX`
+All functions with the naming convention :code:`test_XXX`
 are run. Here we check that the dimensions are correct for the 3D mesh.
 
 If the value is not an integer, you can be subject to floating point errors,
-so :code:`assertTrue` might be too harsh. In this case, you will want to use a
-tolerance. For instance in `test_maps.py <https://github.com/simpeg/simpeg/blob/main/tests/base/test_maps.py>`_
-
+so :code:`assert ==` might be too harsh. In this case, you will want to use
+the ``numpy.testing`` module to check for approximate equals. For instance,
 
 .. code:: python
 
-    class MapTests(unittest.TestCase):
-
-        # method setUp is used to create meshes
-
-        def test_mapMultiplication(self):
-            M = discretize.TensorMesh([2,3])
-            expMap = maps.ExpMap(M)
-            vertMap = maps.SurjectVertical1D(M)
-            combo = expMap*vertMap
-            m = np.arange(3.0)
-            t_true = np.exp(np.r_[0,0,1,1,2,2.])
-            self.assertLess(np.linalg.norm((combo * m)-t_true,np.inf),TOL)
+    import numpy as np
+    import discretize
+    from SimPEG import maps
+
+    def test_mapMultiplication(self):
```

Comment:
do we want to update this to use more pep8-compliant naming? `test_map_multiplication` 

and M --> `mesh` etc?

Reply:
might as well!